$OpenBSD: patch-platform_x11_jstest_main_c,v 1.1 2016/01/19 05:20:51 semarie Exp $

add pledge(2) to mujstest
  - rpath : OPEN command
  - wpath cpath : SCREENSHOT command

--- platform/x11/jstest_main.c.orig	Sun Jan 17 13:50:58 2016
+++ platform/x11/jstest_main.c	Sun Jan 17 13:52:29 2016
@@ -310,6 +310,12 @@ main(int argc, char *argv[])
 	if (fz_optind == argc)
 		usage();
 
+	if (pledge("stdio rpath wpath cpath", NULL) == -1)
+	{
+		fprintf(stderr, "pledge: %s\n", strerror(errno));
+		exit(1);
+	}
+
 	ctx = fz_new_context(NULL, NULL, FZ_STORE_DEFAULT);
 	if (!ctx)
 	{
